README.md

```diff
@@ -16,23 +15,19 @@ This is why I hope for this repository to become a central place to collect addi
 See [CREDIT](CREDIT.csv) for a list of sources and authors of the work contained in this collection as well as the corresponding licences.
 
 ## How to Install
-
 You can add our PPA (Personal Package Archive) to your sources:
 ```
 sudo add-apt-repository ppa:cybre/elementaryplus
 sudo apt-get update
 sudo apt-get upgrade && sudo apt-get install elementaryplus
 ```
-or just download the directory or clone the repository and run `./install.sh` from the parent directory of this repository.
-Then select the theme in [elementary tweaks](https://code.launchpad.net/~versable/elementary-community/elementary-tweaks).
+or just download the directory or clone this repository:
```

Comment:
- Why does this sentence end with a colon?
- Linking "download" to `https://github.com/mank319/elementaryPlus/archive/master.zip` would make it easier for GitHub unexperienced users to find the correct download link.